client/src/app.jsx

```diff
@@ -27,11 +27,15 @@ export const App = () => {
 
   const [mainProduct, setMainProduct] = React.useState("37314");
   const [mainProductName, setMainProductName] = React.useState( "Slacker's Slacks" );
-  const [currentStyleId, setCurrentStyleId] = React.useState('221014');
+  const [currentStyleId, setCurrentStyleId] = React.useState(221014);
 
-  const handleChangeProduct = (info) => {
-    console.log(info.id.toString(), info.name)
+  const handleChangeProduct = (info, style) => {
+    console.log(`-APP-
```

Comment:
If you're done with the console.log, you can delete it or comment it out.

Reply:
thanks.